curp/src/server/raw_curp/log.rs

```diff
@@ -122,8 +140,21 @@ impl<C: 'static + Command> Log<C> {
                 continue;
             }
 
+            let Ok(entry_size) = serialized_size(&entry) else {
```

Comment:
Why do we use a different way to handle serialized_size result? In the above change `expect` is used.

Reply:
Fixed